### PR TITLE
Bump GCP CCM again

### PR DESCRIPTION
**Pull request by upodroid**
@justinsb mentioned some CCM bugs have been squashed since the 26.2.4 release.

We'll pin this to a release version soon

### CODE REVIEW

**Comment by upodroid** [comment]
So the permissions errors have been fixed but the CCM has this bug 

https://github.com/kubernetes/cloud-provider-gcp/issues/612

File: pkg/model/components/gcpcloudcontrollermanager.go [hunk 1]
```diff
@@ -61,11 +61,11 @@ func (b *GCPCloudControllerManagerOptionsBuilder) BuildOptions(options interface
 	}
 
 	if ccmConfig.Controllers == nil {
-		changes := []string{"*,-gkenetworkparamset"}
+		changes := []string{"*"}
 
 		// Turn off some controllers if kops-controller is running them
 		if clusterSpec.IsKopsControllerIPAM() {
-			changes = append(changes, "-nodeipam", "-route")
+			changes = append(changes, "-node-ipam-controller", "-node-route-controller")
```
Comment (upodroid) [inline pkg/model/components/gcpcloudcontrollermanager.go:68]: controllers got renamed recently

```
 mahamed  REDACTED  ~  $  docker run gcr.io/k8s-staging-cloud-provider-gcp/cloud-controller-manager:master --help | grep controllers
      --controllers strings                       A list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller named 'foo', '-foo' disables the controller named 'foo'.
                                                  All controllers: cloud-node-controller, cloud-node-lifecycle-controller, gkenetworkparamset, node-ipam-controller, node-route-controller, service-lb-controller
                                                  Disabled-by-default controllers: gkenetworkparamset (default [*
```

**Comment by upodroid** [comment]
This can be merged now, test failures aren't related this change

**Comment by upodroid** [comment]
It is optional

/skip pull-kops-kubernetes-e2e-cos-gce-slow

**Review by hakman — APPROVED** [review]

**Comment by upodroid** [comment]
Repushed

**Pull request merged**